_templates/navbar.html

```diff
@@ -1,7 +1,6 @@
-<a href="{{pathto('install')}}">Install</a> · 
-<a href="{{pathto('documentation')}}">Docs</a> ·
-<a href="{{pathto('videos')}}">Videos</a> · 
+<a href="http://jupyter.readthedocs.org/en/latest/install.html" target="_blank">Install</a> · 
```

Comment:
Faire enough, but I don't want IPython to point indefinitely to Jupyter, we have to push the identity separation at some point.